### ISSUE TITLE
Lag while typing messages on Android 13

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#890]
Steps to reproduce:

1. Join our existing community using Android 13
2. Send a few messages and wait a while

Expected: characters appear immediately as you type. 

Actual: characters appear after a huge delay as you type. 

This is bad enough as to make the app unusable for me, on the latest hardware. Pixel 6. 

Sending a message also seems to make this bug occur, so we should check this case too. Steps: 1. Send a message 2. Try sending another message. Expected: you can type as normal. Actual: framerate lags.

**Comment by holmesworcester** [comment]
i moved this back to sprint, given that possibly #761 is now unblocked

**Comment by holmesworcester** [comment]
duplicate of #918
